client/src/GlobalStyles.js

```diff
@@ -7,19 +7,21 @@ export default createGlobalStyle`
   --secondary: #354AA8;
   --background: #F2F0F0;
   --font: #4d4c4c;
+  --nav-inact: #D8D8D8;
 }
 
 * {
   box-sizing: border-box;
 }
 
 body {
-  margin: 0;
   font: 0.9rem Helvetica, sans-serif;
+  margin: 0;
 }
 
 main {
   background: var(--background);
   padding: 1rem;
+  height: 100vh;
```

Comment:
Not sure what happens with your `<Header />`component if you set the height of main to 100vh as it's outside your `<main>`? Does this maybe lead to some strange styling?

Maybe create a wrapper or container with 100vh which wraps up all your components?

Reply:
Very well seen, thank you! Since my header and footer have the `position: fixed` I am lucky and the styling is still fine.